loop_wcbench.sh

```diff
@@ -28,8 +31,8 @@ Run WCBench against OpenDaylight in a loop.
 OPTIONS:
     -h Show this help message
     -v Output verbose debug info
-    -l Loop WCBench runs without restarting ODL
-    -r Loop WCBench runs, restart ODL between runs
+    -l Loop WCBench runs for given number of times without restarting ODL
```

Comment:
If we make num_runs optional (which we need to do for backwards compatibility), something like this would be better: "[num_runs] Loop WCBench given number of times without restarting ODL". It also keeps the line to <80 chars.